ampe.c

```diff
@@ -782,8 +796,45 @@ static int plink_frame_tx(
     ies += sizeof(*ht_op);
   }
 
+  if (action != PLINK_CLOSE && mesh->conf->channel_width >= CHAN_WIDTH_80 &&
+      sband->vht_cap.vht_supported) {
+    *ies++ = IEEE80211_EID_VHT_CAPABILITY;
+    *ies++ = sizeof(sband->vht_cap.cap) + sizeof(sband->vht_cap.mcs);
+    memcpy(ies, &sband->vht_cap.cap, sizeof(sband->vht_cap.cap));
+    ies += sizeof(sband->vht_cap.cap);
+    memcpy(ies, &sband->vht_cap.mcs, sizeof(sband->vht_cap.mcs));
+    ies += sizeof(sband->vht_cap.mcs);
+
+    *ies++ = IEEE80211_EID_VHT_OPERATION;
+    *ies++ = 5;
+    vht_op = (struct vht_op_ie *)ies;
+    switch (mesh->conf->channel_width) {
+      case CHAN_WIDTH_80:
+      case CHAN_WIDTH_80P80:
+      case CHAN_WIDTH_160:
+        vht_op->width = 1;
+        break;
+      default:
+        vht_op->width = 0;
```

Comment:
If this is set to 0, does that mean that the rest of the VHT capabilities info is ignored, since we'd then be in HT mode?

Reply:
You can still operate in VHT mode in 20 or 40 MHz; in that case there are additional rates available (256-QAM) using the VHT MCS indexes.  However in this case I only put the VHT caps in the popen frame if the channel width is 80 or up, perhaps that's something worth revisiting.